### PR TITLE
feat(blocks): code block new design

### PR DESCRIPTION
**Pull request by golok727**
Close: BS-114

- Redesigned code block
- Moved the `menu` from database to `_common`

### CODE REVIEW

**Comment by golok727** [comment]
* **#7084** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7084?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @golok727 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by golok727** [comment]
@donteatfriedrice should i hide the ai and caption button ?

File: packages/blocks/src/_common/icons/text.ts [hunk 1]
```diff
@@ -1468,3 +1468,8 @@ export const EmbedEdgelessIcon = html`<svg
     fill="#77757D"
   />
 </svg> `;
+
+export const CommentIcon = svg`<svg width="20" height="20" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (donteatfriedrice) [inline packages/blocks/src/_common/icons/text.ts:1472]: We don't support the comment function yet. The design team should have updated the design. The comment button is currently not needed.

By the way, when we add new icon, we'd better set the fill to 'currentColor', and set the color in icon container, thus the icon color can be followed with the theme.

File: packages/blocks/src/code-block/components/code-actions.ts [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        ${CopyIcon}
+
+        <affine-tooltip tip-position="top" .offset=${5}
+          >Copy Code</affine-tooltip
+        >
+      </icon-button>
+
+      <!-- Not implemented -->
+      <icon-button
+        class="code-action-button hidden"
```
Comment (donteatfriedrice) [inline packages/blocks/src/code-block/components/code-actions.ts:175]: We don't support the comment function yet. The design team should have updated the design. The comment button is currently not needed.

**Review by donteatfriedrice — CHANGES_REQUESTED** [review]

**Pull request closed without merging**